examples/alchemy-daapp/src/surfaces/onboarding/OnboardingController.ts

```diff
@@ -62,29 +62,9 @@ const onboardingStepHandlers: Record<
     if (!context.owner) {
       throw new Error("No connected account or address");
     }
-    return {
-      nextStep: OnboardingStepIdentifier.GET_ENTRYPOINT,
-      addedContext: {},
-    };
-  },
-  // This step gets the entrypoint for the smart account.
-  [OnboardingStepIdentifier.GET_ENTRYPOINT]: async (context) => {
```

Comment:
I think this example is pretty good actually, having it here calls out that this is one way to get the entrypoint address if you don't want to rely on the package being up to date.

on the flip side, removing it does make things simpler for peeps

leave it up to you, but could be worth leaving it here

Reply:
Prefer abstracting out entry point away.